src/modules/PackageSlideShow.rb

```diff
@@ -401,78 +355,52 @@ def InitPkgData(force)
       ResetPackageSummary()
       # Reinititalize some globals (in case this is a second run)
       @total_size_installed = 0
-      # total_time_elapsed  = 0;
-      # start_time    = -1;
       @current_src_no = -1 # 1..n
       @current_cd_no = -1 # 1..n
       @next_src_no = -1
       @next_cd_no = -1
       @last_cd = false
-      @unit_is_seconds = false # begin with package sizes
+      @unit_is_seconds = false # begin without showing time before it stabilize a bit
       @bytes_per_second = 1
 
       src_list = Pkg.PkgMediaNames
-      @inst_src_names = Builtins.maplist(src_list) do |src|
-        Ops.get_string(src, 0, "CD")
-      end
+      @inst_src_names = src_list.map { |src| src[0] || "CD" }
```

Comment:
NP: I usually prefer `.first`...

Reply:
it later use [1] for different. So it is not first element, but index to some internal Pkg list. That is why I keep explicit 0.